CFEScraping/index.js

```diff
@@ -0,0 +1,30 @@
+try {
+    const puppeteer = require('puppeteer');
+    const URL = 'https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/';
+    function OpenBrowser() {
+        (async () => {
```

Comment:
the OpenBrowser should be the async fuction. we will not auto-execute we will execute in other place